docs/source/notebooks/08_Getting_and_bias_correcting_CMIP6_data.ipynb

```diff
@@ -0,0 +1,1788 @@
+{
```

Comment:
<p>I think in general it would be preferable to use average_shape instead of subset_shape, to account for variations in grid cell area across latitude.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/raven/pull/424/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>I agree this would be best, but I can't find the method in Clisops? Maybe another package has this?</p>

Reply:
Damn, I was importing only subset from clisops.core. That is why I couldn't find it...

Reply:
<p>Changed for average_shape. </p>